app/validation.py

```diff
@@ -41,8 +42,12 @@ def load_schemas(schemas_path, schema_names):
 _SCHEMAS = load_schemas(JSON_SCHEMAS_PATH, SCHEMA_NAMES)
 
 
-def get_validator(schema_name):
-    schema = _SCHEMAS[schema_name]
+def get_validator(schema_name, enforce_required=True, required_fields=[]):
```

Comment:
It's a good idea to avoid using mutable objects as default argument values since they're only created once during function object creation and are shared between all calls. The usual workaround is:
```python
def get_validator(..., required_fields=None):
    if required_fields is None:
        required_fields = []
```